README.md

```diff
@@ -25,18 +37,19 @@ const Reader = require('oer-utils/reader')
 const reader = Reader.from(new Buffer('1234', 'hex'))
 
 const v1 = reader.readUInt8()
-const v2 = reader.readUInt8()
+const v2 = reader.readUInt8BigNum()
```

Comment:
What do you think of `readBigUInt8()` for the naming pattern? (slightly shorter, and a little easier to read)

Reply:
I think I prefer it at the end. `BigUInt8` sounds like the UInt8 is big, but that's a bit of a contradiction. It's more like `readUInt8AsBigNumber`, but that seems a little too verbose